### PR TITLE
Wednesday deadline pull request for code review

### PR DESCRIPTION
**Pull request by slavkojos**
This is a pull request indicating we are ready for a a code review from Microverse TSE's

In the commits it is clearly displayed that the webpage is built like this:

1. Added the basic HTML skeleton and imported all of the assets, fonts.
2. The black header and the nav under it is built using flexbox.
3. Cover section (those 3 images on top) are built using css-grid with some media queries.
4. Topics and funding sections are built using css-grid, no media queries.
5. "Latest news" section is built using flexbox and adapted to smaller viewports using media queries.
6. Finally, we focused on polishing the bugs and quirks to refine the final look of the page
7. With those changes, the page is finally published on github pages.

https://slavkojos.github.io/Microverse---Week-2---Responsive-Design/

P.S.  The page is missing "latest deals" section, the plan is to add it tomorrow.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TNW</title>
+</head>
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css"
+    integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous" />
+<link rel="stylesheet" href="styles.css" />
+
```
Comment (bertil291utn) [inline index.html]: ## HTML validator
- Check with HTML validator 
image

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,911 @@
...
+    min-height: 75px;
+  }
+
+  .tile {
+    width: 75px;
+    height: 75px;
+  }
+
+  .story {
+    display: flex;
```
Comment (bertil291utn) [inline styles.css]: ## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.flex{
  display: flex;
}
```
**Html file**

```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish the actual page, you required to do these changes, listed below:
### General requirements
- Fix linter issues
### FILE: README.md
- Remove acknowledgment section if you have nothing  to add
- Live demo is the title and inside that section add the live demo link 
- Add a screenshot 


### Sepecific requirements 

| Section/Screen                             | Large(1024px) | Medium(768px) | Small |
| :----------------------------------------- | :---: | :----: | :---: |
| Navigation bar                             |       |    4    |       |
| Top 3 articles section                     |       |        |       |
| Latest news section                        |  1     |        |       |
| Latest Funding section                     |       |    5    |       |
| Categories (Apps, Gear, Tech, etc) section |    2   |    6    |       |
| Latest deals section                       |    3   |        |       |
| Footer                                     |       |        |       |
###### This table is given each section with his screen width approach, each issue is given by number below this table

1. For this approach screen are four blocks instead of just three

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

2. Instead two columns actually should three

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

3. Add the latest section that is missing 

4. The left logo is missing

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

5.  For this screening approach this section has three blocks

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

6.  For this screening approach, this section has two blocks

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

### Summary
- Add the latest section that is missing 
-  Don't forget to work with inline comments below this comment box.

***

Guys work on the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: styles.css (hunk 2 above)
Comment (slavkojos) [reply styles.css]: We fixed all of the requested changes per first code review. Requesting a new code review

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @ & @**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴
You did a great job implementing the changes required by the last CR. there is one small thing to be changed.
- [ ] make sure that the borders for the icons in the navbar take the whole width of the navbar.
Capture

- [ ] Make sure to increase your usage of semantic html. in this case, use the section tag for every section instead of using the div tag.


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Comment by slavkojos** [comment]
Thank you for your feedback. We implemented the requested changes.

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer :wave:

I just reviewed your project and I am very happy to inform you that your project is approved.
Congratulations on achieving this milestone in your career. :clap:


_I am looking forward to seeing your next project._

Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Pull request merged**